src/test/java/pages/SignInPage.java

```diff
@@ -6,6 +6,9 @@
 import org.openqa.selenium.support.PageFactory;
 
 public class SignInPage {
+    public SignInPage(WebDriver driver) {
```

Comment:
Please add `private final WebDriver driver` variable

Reply:
we aren't using it at this class, do we actually need it?